buildpipeline/Core-Setup-PortableLinux-x64-master.json

```diff
@@ -0,0 +1,154 @@
+{
+  "build": [
+    {
```

Comment:
I'll also log an issue for adopting the Docker cleanup script from buildtools, which can be done later.